site/content/docs/include/deployment.en.md

```diff
@@ -0,0 +1,10 @@
+<div class="separator"></div>
+
+<a id="deployment" href="#deployment" class="field">`deployment`</a> <span class="type">Map</span>  
+The deployment section contains parameters relating to the ECS deployment.
```

Comment:
```suggestion
The deployment section contains parameters to control how many tasks run during the deployment and the ordering of stopping and starting tasks.
```

Reply:
That's specifically for this feature right 🤔 I feel like this parent field should carry broader meaning

Reply:
I copied it from the ECS docs 🤣 https://docs.aws.amazon.com/AmazonECS/latest/developerguide/service_definition_parameters.html#sd-deploymentconfiguration but up to you boss

Reply:
lol that's fair